packages/core/src/invoke.ts

```diff
@@ -38,9 +38,10 @@ export function spawnMachine<
       ) as TMachine;
     }
     const childService = interpret(resolvedMachine, {
-      ...options, // inherit options from this interpreter
+      ...options,
       parent,
-      id: id || resolvedMachine.id
+      id: id || resolvedMachine.id,
+      clock: (parent as any).clock
```

Comment:
do we need to support `clock` option or has it been introduced solely for testing purposes?

Reply:
In V5 we won't need to; I plan on making the clock a standalone service instead of baked-in.

Reply:
k, gonna leave this here - but remove `clock` in a followup PR completely (we'll have to rewrite some tests to achieve that)